packages/@react-spectrum/tree/src/Tree.tsx

```diff
@@ -108,9 +108,7 @@ function TreeItem<T>(props: TreeItemProps<T>) {
           className={linkClassName}>
           {hasChildNodes &&
             <ChevronRightMedium
-              UNSAFE_className={classNames(styles, 'spectrum-TreeView-indicator')}
-              onMouseDown={e => e.stopPropagation()}
-              onClick={() => state.toggleKey(item.key)} />
```

Comment:
These have no effect. I replaced both functions with `console.log` and clicked the chevron in the storybook to verify that these events are not triggered.